test/jasmine/tests/parcoords_test.js

```diff
@@ -1484,7 +1484,7 @@ describe('parcoords constraint interactions', function() {
         .then(done);
     });
 
-    it('@gl will only select one region when multiselect is disabled', function(done) {
+    it('@noCI @gl will only select one region when multiselect is disabled', function(done) {
```

Comment:
This is fine. This test passes fine locally for me, all the `it`s in this `describe`  have a `@noCI` already.